project/Build.scala

```diff
@@ -98,10 +98,12 @@ object Build extends sbt.Build {
           case ScalaJSCrossVersion.binary => previousBinaryCrossVersion
           case crossVersion               => crossVersion
         }
+        val prevExtraAttributes =
+          thisProjectID.extraAttributes.filterKeys(_ != "e:info.apiURL")
```

Comment:
This is actually a fix for the issue I experienced in #2019, which came back hard on this commit for all Scala versions, not just 2.12.x.